src/leds.c

```diff
@@ -261,6 +275,40 @@ static void anim_knight_rider(Leds *leds, const LedStrip *strip, const LedBar *b
     }
 }
 
+static void anim_felony(Leds *leds, const LedStrip *strip, const LedBar *bar, float time) {
+    uint32_t color_off = 0x00000000;
+
+    float state_duration = 0.05f;
```

Comment:
Please use `static const` for the above two constants.

Reply:
Done!